tekton/ci-workspace/templates/all-template.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: TriggerTemplate
+metadata:
+  name: tekton-all-ci-pipeline
+spec:
+  params:
+  - name: buildUUID
+    description: UUID used to track a CI Pipeline Run in logs
+  - name: package
+    description: org/repo
+  - name: pullRequestNumber
+    description: The pullRequestNumber
+  - name: pullRequestUrl
+    description: The HTML URL for the pull request
+  - name: gitRepository
+    description: The git repository that hosts context and Dockerfile
+  - name: gitRevision
+    description: The Git revision to be used.
+  - name: gitCloneDepth
+    description: Number of commits in the change + 1
+  - name: gitHubCommand
+    description: |
+      The GitHub command that was used a trigger. This is only available when
+      this template is triggered by a comment. The default value is for the
+      case of a pull_request event.
+    default: ""
+  - name: labels
+    description: List of labels currently on the Pull Request
+  resourcetemplates:
+  - apiVersion: tekton.dev/v1beta1
+    kind: PipelineRun
+    metadata:
+      name: check-pr-labels-$(uid)
+      labels:
+        prow.k8s.io/build-id: $(tt.params.buildUUID)
+        tekton.dev/check-name: check-pr-has-kind-label
+        tekton.dev/kind: ci
+        tekton.dev/pr-number: $(tt.params.pullRequestNumber)
+      annotations:
+        tekton.dev/gitRevision: "$(tt.params.gitRevision)"
+        tekton.dev/gitURL: "$(tt.params.gitRepository)"
+    spec:
+      pipelineRef:
+        name: tekton-kind-label
+      params:
+        - name: labels
+          value: $(tt.params.labels)
+        - name: checkName
+          value: check-pr-has-kind-label
+        - name: gitHubCommand
+          value: $(tt.params.gitHubCommand)
+      resources:
+        - name: source
+          resourceSpec:
+            type: git
+            params:
+            - name: revision
+              value: master
```

Comment:
So according to https://github.com/tektoncd/plumbing/issues/681, community will be using `main` starting tomorrow or the day after, and slowly the rest of the projects 🙃